stanford_news.features.field_instance.inc

```diff
@@ -31,6 +31,13 @@ function stanford_news_field_default_field_instances() {
         'type' => 'text_default',
         'weight' => 4,
       ),
+      'stanford_extras_single_region' => array(
```

Comment:
I tested this on a plain jane JSA and I was surprised and delighted to find that this did not actually mark the feature as overridden. 

A maintenance problem comes up with future work on the news feature. If someone without all of the required modules re-exports this module much of it will be lost. This is not new to us and good thing we have version control.